api/src/test/java/org/pmiops/workbench/utils/TestMockFactory.java

```diff
@@ -385,32 +381,6 @@ public static void assertEqualDemographicSurveys(
     assertThat(normalizeLists(survey1)).isEqualTo(normalizeLists(survey2));
   }
 
-  public static LeonardoListPersistentDiskResponse createLeonardoListPersistentDiskResponse(
-      String pdName,
-      LeonardoDiskStatus status,
-      String date,
-      String googleProjectId,
-      DbUser user,
-      @Nullable AppType appType) {
-    LeonardoListPersistentDiskResponse response =
-        new LeonardoListPersistentDiskResponse()
-            .name(pdName)
-            .size(300)
-            .diskType(LeonardoDiskType.STANDARD)
-            .status(status)
-            .auditInfo(new LeonardoAuditInfo().createdDate(date).creator(user.getUsername()))
-            .cloudContext(
-                new LeonardoCloudContext()
-                    .cloudProvider(LeonardoCloudProvider.GCP)
-                    .cloudResource(googleProjectId));
-    if (appType != null) {
-      Map<String, String> label = new HashMap<>();
-      label.put(LEONARDO_LABEL_APP_TYPE, appTypeToLabelValue(appType));
```

Comment:
looks like we're injecting some label in the response here...we don't need this anymore?

Reply:
This method was only for the old Leo client.  See the new-client equivalent below this.  We retain the labeling there.